### DISCUSSION TITLE
Implement "States"

### DISCUSSION THREAD

**Discussion by joepv** [alexkn/node-red-contrib-homeconnect#6] [Q&A] [answered]
Before I send a command to my Bosch dishwasher I want to check the [Remote Start Allowance State](
https://developer.home-connect.com/docs/api/status/remotestartallowancestate). Else the request is not handled by the dishwasher.

Also the [Door State](https://developer.home-connect.com/docs/status/door_state) is interesting, because you can check if the appliance door is open and act on that.

Is it possible to add a _States_ Node?
Documentation is found here:
https://developer.home-connect.com/docs/api/status/remotecontrolactivationstate

**Comment by alexkn** [comment, marked as answer]
It's already possible with a request node with the following settings:
Tag: Status
Action: get_status_value
Status Key: BSH.Common.Status.RemoteControlStartAllowed

**Comment by joepv** [comment]
Thank you, I didn't know that. I'll close the ticket.
